releases/release-8.3.0.md

```diff
@@ -0,0 +1,409 @@
+
+### Configuration file parameters
+
+| Configuration file | Configuration parameter | Change type | Description |
+| -------- | -------- | -------- | -------- |
+| TiDB | [`tikv-client.batch-policy`](/tidb-configuration-file.md#batch-policy-new-in-v830) | Newly added | Controls the batching strategy for requests from TiDB to TiKV. |
+| PD   |  [`security.redact-info-log`](/pd-configuration-file.md#redact-info-log-new-in-v50) |  Modified | Support setting the value of the PD configuration item `security.redact-info-log' to `marker` to mark sensitive information in the log with single-angle quotation marks `‹›` instead of shielding it directly. With the `marker` option, you can customize the redaction rules.  |
+| PD   | [`schedule.max-merge-region-keys`](/pd-configuration-file.md#max-merge-region-keys) |  Modified | Change the default value from `200,000` to `540,000` to accommodate the default value change of `schedule.max-merge-region-size`. This change helps reduce management pressure caused by an excessive number of Regions as the cluster size grows. |
+| PD   | [`schedule.max-merge-region-size`](/pd-configuration-file.md#max-merge-region-size) |  Modified | Change the default value from `20` MiB to `54` MiB to accommodate the increase in the TiKV default Region size to `256` MiB. This change helps reduce management pressure caused by an excessive number of Regions as the cluster size grows. |
+| TiKV  | [`security.redact-info-log`](/tikv-configuration-file.md#redact-info-log-new-in-v408)  | Modified | Support setting the value of the TiKV configuration item `security.redact-info-log' to `marker` to mark sensitive information in the log with single-angle quotation marks `‹›` instead of shielding it directly. With the `marker` option, you can customize the redaction rules.  |
+| TiKV  | [`coprocessor.region-split-size`](/tikv-configuration-file.md#region-split-size)  | Modified | Change the default value from `96` MiB to `256` MiB. This change helps reduce management pressure caused by an excessive number of Regions as the cluster size grows. |
+| TiKV  | [`coprocessor.region-max-size`](/tikv-configuration-file.md#region-max-size)  | Modified | Change the default value from `144` MiB to `384` MiB to accommodate the increase in `coprocessor.region-split-size` to `256` MiB. |
+| TiKV  | [`backup.sst-max-size`](/tikv-configuration-file.md#sst-max-size)  | Modified | Change the default value from `144` MiB to `384` MiB to accommodate the increase in `coprocessor.region-max-size` to `384` MiB. |
```

Comment:
```suggestion
| TiKV  | [`security.redact-info-log`](/tikv-configuration-file.md#redact-info-log-new-in-v408)  | Modified | Support setting the value of the TiKV configuration item `security.redact-info-log' to `marker` to mark sensitive information in the log with single-angle quotation marks `‹›` instead of shielding it directly. With the `marker` option, you can customize the redaction rules.  |
```